### PR TITLE
Fixed issue #331

### PR DESCRIPTION
**Pull request by Sartech-B**
solved issue #331 and added new tile!

### CODE REVIEW

**Comment by duttaditya18** [comment]
@Sartech-B #332 solved #331 before you. I even mentioned it in the issue that I am working on it.

**Comment by Sartech-B** [comment]
but his is not completely fixed
some characters are not visible

**Comment by naveentherocker** [comment]
@Sartech-B Please Close your Pull Request because @duttaditya18 has claimed it before you and You can't work on #331 without his permission because he is the owner of that issue.

File: js/tiles.js [hunk 1]
```diff
@@ -416,6 +416,33 @@ const door = {
             this.ground.show();
         }
     }),
+    //mine
+    vector_pattern: Object.assign({}, OpenDoors, {
```
Comment (niccokunzmann) [inline js/tiles.js]: You would write vectorPattern in JavaScript.

**Comment by niccokunzmann** [comment]
Thanks!

**Pull request merged**